packages/ui-flex/src/Flex/index.tsx

```diff
@@ -88,7 +88,8 @@ class Flex extends Component<FlexProps> {
   }
 
   renderChildren(children: FlexProps['children']) {
-    return Children.map(children, (child) => {
+    // TODO this might fail if children are () => ReactNode
```

Comment:
React 18 has better types for some things, if it uncovered a potential bug I've left a comment.